Sources/Vitamin/Components/Progressbar/VitaminProgressbar.swift

```diff
@@ -387,23 +400,33 @@ extension VitaminProgressbar {
     // change the cuurent progress of the progress bar
     // has no effect in `.indeterminate` progress type
     private func applyProgress(_ newProgress: CGFloat) {
-        if case .circular(let circularSize, _) = self.variant {
+        if case .circular(let circularSize, let style) = self.variant {
             if progressType != .indeterminate {
                 self.progressLayer.strokeEnd = newProgress
+
                 self.percentageLabel.attributedText = "\(displayableProgress())".styled(
                     as: circularSize.textStyle,
                     with: labelColor)
                 self.configLabels()
+                switch style {
+                    case .percentage:
+                        accessibilityValue = displayableProgress()
+                case .empty, .image:
+                    break
+                }
```

Comment:
I think the value should also be indicated in `.empty` and `.image` case, because the value is still indicated by the track of the progress bar, even if it is not displayed as a figure.
```suggestion
                accessibilityValue = displayableProgress()
```